wasmtime/_exportable.py

```diff
@@ -0,0 +1,12 @@
+__all__ = (
+    "Exportable",
+)
```

Comment:
Could this avoid the usage of `__all__` while the other PR is sorted out?

Reply:
Sure.